public/components/create_report/delivery/delivery.tsx

```diff
@@ -13,60 +13,202 @@
  * permissions and limitations under the License.
  */
 
-import React from 'react';
+import React, { useState } from 'react';
 import {
-    EuiFieldText,
-    EuiFormRow,
-    EuiPageHeader,
-    EuiTitle,
-    EuiPageContent,
-    EuiPageContentBody,
-    EuiHorizontalRule,
-    EuiTextArea,
-  } from '@elastic/eui';
+  EuiFieldText,
+  EuiFormRow,
+  EuiPageHeader,
+  EuiTitle,
+  EuiPageContent,
+  EuiPageContentBody,
+  EuiHorizontalRule,
+  EuiTextArea,
+  EuiCheckbox,
+  EuiSpacer,
+  EuiComboBox,
+  EuiText,
+  EuiLink,
+  EuiPopover,
+  EuiListGroup,
+} from '@elastic/eui';
 import { DeliveryRecipientsBox } from './delivery_recipients_box';
-import { DeliveryChannelRadio } from './delivery_channel_radio';
+import { email_recipient_options } from './delivery_constants';
 
-const ReportDelivery = (props) => {
-  const { 
-    deliveryEmailSubject, 
-    onChangeDeliveryEmailSubject,
-    deliveryEmailBody,
-    onChangeDeliveryEmailBody
-  } = props;
+const insert_placeholder_options = [
+  {
+    label: 'Report details URL',
```

Comment:
Do you really need prefix `Report` ?

Reply:
Prefix `Report` is included in the UX mock-ups